components/scream/src/physics/shoc/shoc_iso_c.f90

```diff
@@ -497,4 +497,18 @@ subroutine linear_interp_c(x1,x2,y1,y2,km1,km2,ncol,minthresh) bind (C)
 
   end subroutine linear_interp_c  
 
+ subroutine shoc_diag_second_moments_lbycond_c(shcol, wthl, wqw, uw, vw,ustar2, wstar, &
+            wthlo, wqwo, uwo, vwo, wtkeo, thlo, qwo, qwthlo) bind(C)
+   use shoc, only: diag_second_moments_lbycond
+
+   ! argmens
```

Comment:
Let's just get rid of these kinds of comments.

Reply:
done